src/shaders/line.vertex.glsl

```diff
@@ -102,7 +104,8 @@ void main() {
     mediump vec2 offset2 = offset * a_extrude * EXTRUDE_SCALE * normal.y * mat2(t, -u, u, t);
 
     vec4 projected_extrude = u_matrix * vec4(dist * u_pixels_to_tile_units, 0.0, 0.0);
-    gl_Position = u_matrix * vec4(pos + offset2 * u_pixels_to_tile_units, 0.0, 1.0) + projected_extrude;
+    v_projected_pos = u_matrix * vec4(pos + offset2 * u_pixels_to_tile_units, 0.0, 1.0) + projected_extrude;
```

Comment:
Should you add `DEPTH_OCCLUSION` guards here as well so that the change would be no-op unless explicitly enabled?